lib/pc_api.h

```diff
@@ -479,4 +481,7 @@ uint32_t pc_patch_is_sorted(const PCPATCH *pa, const char **name, int ndims, cha
 /** Subset batch based on index */
 PCPATCH* pc_patch_range(const PCPATCH *pa, int first, int count);
 
+/** assign a schema to the patch */
+PCPATCH *pc_patch_set_pcid(PCPATCH *patch, const PCSCHEMA *schema, double defaultvalue);
```

Comment:
Thinking back, I would call this function `pc_patch_set_schema` as it is what it actually does (while leaving setpcid at the sql level)